test/integration/rest.test.js

```diff
@@ -40,6 +40,7 @@ describe('REST client authentication', () => {
 
   after(done => {
     server.close(done);
```

Comment:
Well it used to be. Let's ship this and I'll check it out. I have some client updates to push soon anyway. Just working through them in a project.